ddtrace/tracer/span.go

```diff
@@ -203,7 +222,7 @@ func (s *span) Root() Span {
 // As opposed to the public Root method, this one returns the actual span type
 // when internal usage requires it (to avoid type assertions from Root's return
 // value).
-func (s *span) root() *span {
+func (s *Span) root() *Span {
```

Comment:
This is short enough... I'm not sure I understand why it's not the exported `Root()` function?

Reply:
Yes, this should be exportable now. This is an oversight during the refactoring.